Src/DependencyCollector/Net45.Tests/packages.config

```diff
@@ -3,4 +3,5 @@
   <package id="Microsoft.ApplicationInsights" version="2.0.0-beta4" targetFramework="net451" />
```

Comment:
Gitlink only has target of net45. We're not referencing it in code anyway, just using the .exe from the command line.